### ISSUE TITLE
Support of `fileNameIndex` and `params` options by Angular 6+

### ISSUE THREAD

**Issue by psolom** [kzrfaisal/angular-file-uploader#78]
I'm on the Angular 7 and use angular-file-uploader v5.0.2, but it is missing `fileNameIndex` and `uploadAPI.params` configuration options. I would greatly thank you if you add these option and release v5.0.3.

Thanks in advance

**Comment by kzrfaisal** [comment]
Hey @psolom ,
You can upgrade to 6+ version to get these options, there's not much of a difference, if any problem comes there, I'm here to help.

**Comment by psolom** [comment]
I've got error on v6+
```
./node_modules/angular-file-uploader/fesm5/angular-file-uploader.js 9:39-57
"export 'ɵɵdefineInjectable' was not found in '@angular/core'
```

**Comment by kzrfaisal** [comment]
Let me check

**Comment by kzrfaisal** [comment]
Hey @psolom ,
It's fixed now, download the updated version 6.2.7, which is containing the fix.

**Comment by psolom** [comment]
Thanks a lot!
